keys/employees-keys/README.md

```diff
@@ -314,17 +264,35 @@ you need the following in `~/.gitconfig`:
 
 Make sure your
 [Github](https://docs.github.com/en/github/authenticating-to-github/telling-git-about-your-signing-key#telling-git-about-your-gpg-key-2)
-and [Gitlab](https://docs.gitlab.com/ee/user/project/repository/gpg_signed_commits/index.html#adding-a-gpg-key-to-your-account) is set up correctly to provide GPG commit verification.
+and
+[Gitlab](https://docs.gitlab.com/ee/user/project/repository/gpg_signed_commits/index.html#adding-a-gpg-key-to-your-account)
+is set up correctly to provide GPG commit verification.
 
 
 ### Create a pull request with key
 
-After everything is set up, please issue a pull request to this repo with signed
-`your-name-key-signed.asc`.
+After everything is set up, please issue a pull request to this repo with
+signed `your-name-key-signed.asc`.
 
-### FAQ
+### Upload to keys.opengpg.org
 
-#### What to do when this key expires?
+* Go to https://keys.openpgp.org.
+* Upload Your Key:
+  * On the main page, you'll see an "Upload" section.
+  * Click on the "Choose File" button and select the your.name@3mdeb.com.asc
+    file you just created.
+  * Click on the "Upload" button.
+* Verify Your Email Address:
+  * https://keys.openpgp.org will send you an email to verify that the email
+    address associated with the key is valid.
+  * Open the email and click on the verification link.
+  * Once verified, your key will be added to the keys.openpgp.org keyserver.
+* After uploading and verifying, you can search for your key on the keyserver
+  using your email address to ensure it's been uploaded correctly.
+
+## FAQ
+
+### What to do when this key expires?
```

Comment:
This should also mention that renewed keys have to be sent to GitLab and GitHub. However, as I learned yesterday, you can't just upload new key with the same key ID to GitHub, it will be silently ignored. You have to remove old one and then import the new one. I don't know if GitLab has the same problem.

Reply:
@krystian-hebel, thanks for the comment. I tried to address it here da0880b8de12d32c49d1f3637f30cad16bafaee1
If you think something should be improved, please provide an actionable description of what can be changed.

Please don't bother about spelling. The automatic spellchecker should fix that. I cannot afford your rate for spellchecking.